usaspending_api/accounts/v2/filters/account_download.py

```diff
@@ -187,10 +186,6 @@ def generate_gross_outlay_amount_derived_field(filters, account_type):
 def generate_treasury_account_query(queryset, account_type, tas_id, filters):
     """ Derive necessary fields for a treasury account-grouped query """
     derived_fields = {
-        "allocation_transfer_agency_identifier_name": get_agency_name_annotation(
```

Comment:
Looks like removal of `get_agency_name_annotation` what was the reason and benefit for this? And what goes in its place if those fields are still needed?

Reply:
I think i see now -- this was an artifact of needing to do a subquery in django to join `cgac` table?

Reply:
> this was an artifact of needing to do a subquery in django to join cgac table?

Correct. It was performing the CGAC subquery and is no longer needed now that we handle that in SQL.